model/src/test/kotlin/utils/FindingsMatcherTest.kt

```diff
@@ -86,6 +86,59 @@ class FindingsMatcherTest : WordSpec() {
             }
         }
 
+        "Given a license finding in a license file and a file with copyright and license findings, match" should {
+            "not associate that copyright with the root license" {
+                setupLicenseFinding(license = "some id", path = "a/LICENSE")
+                setupLicenseFinding(license = "some other id", path = "some/other/file")
+                setupCopyrightFinding(statement = "some stmt", path = "some/other/file")
+
+                val result = matcher.match(licenseFindings, copyrightFindings)
+
+                result.getFindings("some id").copyrights should beEmpty()
+            }
+        }
+
+        "Given license findings in two license files a copyright finding in a file without license, match" should {
+            "associate that copyright with excatly one root license" {
+                setupLicenseFinding(license = "some id", path = "a/LICENSE")
+                setupLicenseFinding(license = "some other id", path = "b/LICENSE")
+
+                setupCopyrightFinding(statement = "some stmt", path = "some/file")
+
+                val result = FindingsMatcher(LicenseFileMatcher("a/LICENSE", "b/LICENSE"))
+                    .match(licenseFindings, copyrightFindings)
+
+                result.size shouldBe 2
+                result.getFindings("some id").copyrights.map { it.statement } shouldBe listOf("some stmt")
+                result.getFindings("some other id").copyrights.map { it.statement } shouldBe listOf("some stmt")
+            }
+        }
+
+        "Given license findings with 6 copyright findings directly above, match" should {
+            "associate all except the top copyright finding" {
+                // TODO: change the behavior of the algorithm so that also the top most copyright is matched
```

Comment:
Looking at the failed tests in Travis it seems this is already the case.

Reply:
I had to change this test as it didn't pass as is here.

Reply:
This comment can be removed then.

Reply:
I think the comment is still valid. Why do you think it can be removed?

Reply:
That was a misunderstanding.